### PR TITLE
Create navigation between data dashboard an data explorer

### LINKED ISSUE #56: Create navigation between data selection and visualisations
### Description

Using `react-router` one should be able to navigate between the two pages as specified in the design kindly provided by @IT2901-SINTEF01/ux-design.

### Subtasks
- [x] Create routes for both pages, and setup 404 handler with page
- [x] Make sure the URL doesn't contain any octothorpes, e.g. `localhost:3000/#/path/to/resource`. This is a sign of misconfiguration

### Related sources
- Blocked by #53 
- [react-router-dom](https://reactrouter.com/web/guides/quick-start)

### PR DESCRIPTION
**Pull request by emilom**
Added routing with react-router-dom in App. Changed behaviour of AddComponentButton so it routes to data explorer page.
Added 404 page 💀
closes #56

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -5,9 +5,9 @@ html {
...
     margin: 0;
-    min-height: 100%;
+    height: 100vh;
 }
 
 #root {
-    min-height: 100%;
-}
+    height: 100vh;
+}
```
Comment (blauks) [inline src/index.css]:
```suggestion
}

```

File: src/App.tsx [hunk 2]
```diff
@@ -13,7 +16,13 @@ const client = new ApolloClient({
 const App: React.FC = () => {
     return (
         <ApolloProvider client={client}>
-            <Dashboard />
+            <BrowserRouter>
+                <Switch>
+                    <Route path="/explore" component={DataExplorer} />
```
Comment (blauks) [inline src/App.tsx]: Should we have exact path here aswell?

Comment (emilom) [reply src/App.tsx]: Good idea, might be handy later for nested paths. We work great as a team 🥭🥭😏

Comment (blauks) [reply src/App.tsx]: Thanks man

File: src/components/atoms/AddComponentButton.tsx [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import { Card, Text } from 'evergreen-ui';
-import { dashboardItemsVar } from '../../cache';
-import { WEATHER_MET_API } from '../../queries/metApi';
+import { useHistory } from 'react-router-dom';
 
 const AddComponentButton: React.FC = () => {
+    const history = useHistory<string>();
```
Comment (FredrikAugust) [inline src/components/atoms/AddComponentButton.tsx]:
```suggestion
    const history = useHistory();
```

File: src/components/organisms/NotFound.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Pane, Text, Heading } from 'evergreen-ui';
+
+const NotFound: React.FC = () => {
+    return (
+        <Pane display="flex" height="100%" justifyContent="center" alignItems="center" flexDirection="column">
+            <Heading>404</Heading>
+            <Text>Oops! Her ser det ut som det ble noe krøll.. Vi finner ikke siden du leter etter.</Text>
```
Comment (Kpaubert) [inline src/components/organisms/NotFound.tsx:8]: Ellipsis' are usually written with [three](https://en.wikipedia.org/wiki/Ellipsis) dots.

File: src/components/atoms/AddComponentButton.tsx [hunk 5]
```diff
@@ -1,11 +1,12 @@
...
+
     const addDashboardItem = () => {
-        dashboardItemsVar([...dashboardItemsVar(), { size: 3, query: WEATHER_MET_API }]);
+        history.push('/explore');
```
Comment (FredrikAugust) [inline src/components/atoms/AddComponentButton.tsx]: Don't use `useHistory` unless it's needed (programmatic redirection). Stick to links to help screen readers, accessibility &c.

Comment (FredrikAugust) [reply src/components/atoms/AddComponentButton.tsx]: Wrap it in a `Link`.

**Review by FredrikAugust — CHANGES_REQUESTED** [review]

**Review by Kpaubert — COMMENTED** [review]
All else looks good.

File: src/components/organisms/NotFound.tsx (hunk 4 above)
Comment (FredrikAugust) [reply src/components/organisms/NotFound.tsx:8]: They are not actually three dots, it's its own unicode character.

…

You can copy the symbol above.

Comment (blauks) [reply src/components/organisms/NotFound.tsx:8]: Actually...

**Review by FredrikAugust — APPROVED** [review]

File: src/components/organisms/NotFound.tsx (hunk 4 above)
Comment (Kpaubert) [reply src/components/organisms/NotFound.tsx:8]: Refer to the link in my previous comment @FredrikAugust

**Pull request merged**